src/lm_buddy/jobs/__init__.py

```diff
@@ -1,9 +1,15 @@
 from lm_buddy.integrations.wandb import ArtifactLoader, WandbArtifactLoader
-from lm_buddy.jobs._entrypoints import run_finetuning, run_lm_harness, run_simple
+from lm_buddy.jobs._entrypoints import (
+    run_finetuning,
+    run_lm_harness,
```

Comment:
the same question here as above, we now have finetuning and eval that are on the same level, and harness and prometheus that are a subset of eval, do we want to handle in a nested hierarchy?